src/components/shared/Dropdown.css

```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
   background-color: var(--search-overlays-semitransparent);
 }
 
+.dropdown li img.prettyPrint {
+  -webkit-mask: url(/images/prettyPrint.svg) no-repeat;
+  mask: url(/images/prettyPrint.svg) no-repeat;
+  -webkit-mask-size: 100%;
+  mask-size: 100%;
+  background: var(--theme-highlight-blue);
+  width: 13px;
+  height: 13px;
+  margin-right: 5px;
+  vertical-align: middle;
+}
+
+.dropdown li img.blackBox {
```

Comment:
the styles here are largely the same, i think we can factor the similarity out into its own class, `.dropdown--icon` with two modifications `.dropdown--icon_blackbox` and `.dropdown--icon_prettyprint`

Reply:
using combined `.dropdown-icon` that holds the majority of the css + `.blackBox`, `.prettyPrint` & `.file`. It seems that you cannot move the mask size declaration to a different class than declaring the actual mask, interesting.